config/dynamic.json

```diff
@@ -24,11 +24,22 @@
 			},
 			"protected_roles": [
```

Comment:
This and other antags are all done through config, is the TG edits worth editing things that should be done in config instead?